app/package.json

```diff
@@ -10,6 +10,7 @@
   },
   "license": "BSD-2-Clause",
   "dependencies": {
+    "keytar": "^4.0.3",
```

Comment:
`yarn add` does have a flag for that: https://yarnpkg.com/lang/en/docs/cli/add/#toc-yarn-add-exact-e